code/_onclick/hud/new_player.dm

```diff
@@ -208,7 +209,14 @@
 			SSticker.queued_players += new_player
 			to_chat(new_player, span_notice("You have been added to the queue to join the game. Your position in queue is [SSticker.queued_players.len]."))
 		return
-	new_player.LateChoices()
+
+	if(!LAZYACCESS(params2list(params), CTRL_CLICK))
```

Comment:
I think instead, you should make the latejoin menu send an act on its first mount. If you don't receive that after n seconds, spit this message.

Reply:
As in rather than display it every time you click, which will be noise 99% of the time.

Reply:
I could also just make it only show if you click the button more than once, or something. I don't trust myself to make a detection system for this that is more reliable than TGUI itself, especially in dm.

Reply:
I've made an attempt, and it seems to work well enough, so there.